src/app/zap-templates/zcl/data-model/chip/air-quality-cluster.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <name>Air Quality</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes for reporting air quality classification</description>
+    <code>0x005B</code>
+    <define>AIR_QUALITY_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="AIR_QUALITY" type="AirQualityEnum" min="0" max="6" writable="false" isNullable="false" default="0" optional="false">AirQuality</attribute>
+  </cluster>
+
+  <!-- Feature Map -->
+  <bitmap name="Features" type="BITMAP32">
```

Comment:
```suggestion
  <bitmap name="Feature" type="BITMAP32">
```
Sorry for the back-and-forth here, but this probably reads better in terms of the resulting APIs.